data.py

```diff
@@ -242,6 +252,11 @@ def load_sensor_data(self, data: str) -> None:
             elif sensor_name in self.data.keys():
                 self.data[sensor_name].update(sensor_value)
 
+    def load_voltage_data(self, data: str) -> None:
+        # voltage_data = loads(data)
+        voltage_data = loads('{"voltage":7.1312}')
```

Comment:
Swap this out to use the real data.

FWI, this doesn't mean you can't test it, use
```bash
mosquitto_pub -t "status/camera/primary/battery" -m '{"voltage":7.8}'
```